src/pages/applications.tsx

```diff
@@ -4,17 +4,19 @@ import axios from 'axios';
 import { useRouter } from 'next/router';
 import Image from 'next/image';
 
-enum StatusEnum {
-  APPLIED = 'Applied',
-  REJECTED = 'Rejected',
-  PHONE_SCREEN = 'Phone Screen',
-  INTERVIEW = 'Interview',
-  JOB_OFFER = 'Job Offer',
-}
-enum SourceEnum {
-  LinkedIn = 'LinkedIn',
-  Seek = 'Seek',
-}
+const Status = {
+  APPLIED: 'Applied',
+  REJECTED: 'Rejected',
+  PHONE_SCREEN: 'Phone Screen',
+  INTERVIEW: 'Interview',
+  JOB_OFFER: 'Job Offer',
+} as const;
+
+const Source = {
+  LinkedIn: 'LinkedIn',
+  Seek: 'Seek',
+} as const;
+
```

Comment:
This one was my bad. I completely missed that you have to extract the type from the `as const` object.

This was causing some type errors in the typedefs below.

Can you make the following changes and fix the type definitions for the interfaces?

```TSX
const STATUS = {
  APPLIED: "Applied",
  REJECTED: "Rejected",
  PHONE_SCREEN: "Phone Screen",
  INTERVIEW: "Interview",
  JOB_OFFER: "Job Offer",
} as const;

type Status = keyof typeof STATUS;

const SOURCE = {
  LinkedIn: "LinkedIn",
  Seek: "Seek",
} as const;

type Source = keyof typeof STATUS;
```

Reply:
I completely missed the other enums in the other files as well.

Can you please export these so that we can share the typedefs? 

Suggestion: maybe move this into a types file somewhere in the codebase.